packages/web-react/src/components/ScrollView/useScrollViewStyleProps.ts

```diff
@@ -25,8 +27,8 @@ export const useScrollViewStyleProps = ({
   isScrollbarDisabled,
   isScrolledAtEnd,
   isScrolledAtStart,
-  overflowDecorators,
-}: UseScrollViewStyleProps): UseScrollViewStyleReturn => {
+  overflowDecorators = SCROLL_VIEW_DEFAULT_OVERFLOW_DECORATOR,
+}: Partial<UseScrollViewStyleProps>): UseScrollViewStyleReturn => {
```

Comment:
All props set as optional? Why?

Reply:
I was just a little bit confused because this construct changes all the props to `optional`, right? Anyway, we use this hook for internal purpose only, so probably we don't have to deal with it.